README.md

```diff
@@ -1,5 +1,108 @@
-# Run ansible scripts
+# WSO2 Identity Server Ansible scripts
+
+This repository contains the Ansible scripts for installing and configuring WSO2 Identity Server.
+
+## Supported Operating Systems
+
+- Ubuntu 16.04 or higher
+
+## Supported Ansible Versions
+
+- Ansible 2.0.0.2
+
+## Directory Structure
+```
+.
+├── dev
+│   ├── group_vars
+│   │   └── is.yml
+│   ├── host_vars
+│   │   └── wso2is.yml
+│   └── inventory
+├── FAQ.md
+├── files
+│   ├── mysql-connector-java-5.1.45-bin.jar
+│   └── wso2is-linux-installer-x64-5.6.0.deb
+├── issue_template.md
+├── LICENSE
+├── pull_request_template.md
+├── README.md
+├── roles
+│   └── wso2is
+│       ├── tasks
+│       │   ├── customize.yml
+│       │   └── main.yml
+│       └── templates
+│           ├── carbon-home
+│           │   ├── bin
+│           │   │   └── wso2server.sh.j2
+│           │   └── repository
+│           │       ├── conf
+│           │       │   ├── axis2
+│           │       │   │   └── axis2.xml.j2
+│           │       │   ├── carbon.xml.j2
+│           │       │   ├── datasources
+│           │       │   │   └── master-datasources.xml.j2
+│           │       │   ├── identity
+│           │       │   │   └── identity.xml.j2
+│           │       │   ├── registry.xml.j2
+│           │       │   ├── tomcat
+│           │       │   │   └── catalina-server.xml.j2
+│           │       │   └── user-mgt.xml.j2
+│           │       └── deployment
+│           │           └── server
+│           │               └── eventpublishers
+│           │                   ├── IsAnalytics-Publisher-wso2event-AuthenticationData.xml.j2
+│           │                   ├── IsAnalytics-Publisher-wso2event-RoleData.xml.j2
+│           │                   ├── IsAnalytics-Publisher-wso2event-SessionData.xml.j2
+│           │                   └── IsAnalytics-Publisher-wso2event-UserData.xml.j2
+│           └── wso2is.service.j2
+└── site.yml
+```
+
+## Packs to be Copied
+
+Copy the following files to `ansible-is/files` directory.
+
+1. WSO2 Identity Server 5.6.0 package (deb or rpm)
+2. mysql-connector-java-5.1.45-bin.jar
+
+## Running WSO2 Identity Server Ansible scripts
+
+### 1. Run the existing scripts without customization
+The existing Ansible scripts contain the configurations to set-up a single node WSO2 Identity Server pattern. In order to deploy the pattern, you need to replace the `[ip_address]` given in the `inventory` file under `dev` folder by the IP of the location where you need to host the Identity Server. An example is given below.
+```
+[is]
+wso2is ansible_host=172.28.128.4
+```
+
+Run the following command to run the scripts.
+
 `ansible-playbook -i dev site.yml`
 
+If you need to alter the configurations given, please change the parameterized values in the yaml files under `group_vars` and `host_vars`.
+
+### 2. Customize the WSO2 Ansible scripts
+
+The templates that are used by the Ansible scripts are in j2 format in-order to enable parameterization.
```

Comment:
Do we need this information? Are there any other template formats?

Reply:
@Mirage20 the purpose of the README documentation of installation artifacts is to allow someone with zero knowledge about each platform, to easily setup the artifacts and also, to gain a basic understanding on used features of the platform.

Hence, I don't believe this information is redundant although, j2 format is the known template format.

Reply:
+1 for keeping this piece of information.